android/src/main/kotlin/com/onegini/mobile/sdk/flutter/helpers/SdkError.kt

```diff
@@ -0,0 +1,56 @@
+package com.onegini.mobile.sdk.flutter.helpers
+
+import com.onegini.mobile.sdk.flutter.OneWelcomeWrapperErrors
+import io.flutter.plugin.common.MethodChannel
+import okhttp3.Response
+
+class SdkError(
+    private val title: String = "Error",
```

Comment:
I did a quick check on the iOS side and it doesn't appear to be used. So let's remove it for both Android **and** iOS

Reply:
was aware of it but thought it could be useful for a full error details object in case we wanted to make use of it in the future, where the error details object can be used to just throw in alerts. But I think I agree to remove it for now, so will do

Reply:
done